### PR TITLE
[FIAM] Firelog to clearcut migration

### PR DESCRIPTION
**Pull request by eldhosembabu**
[FIAM] Firelog to clearcut migration

### CODE REVIEW

File: firebase-inappmessaging/firebase-inappmessaging.gradle [hunk 1]
```diff
@@ -106,7 +106,7 @@ dependencies {
     implementation 'com.google.firebase:firebase-components:17.1.0'
     implementation project(':protolite-well-known-types')
     implementation 'com.google.android.datatransport:transport-api:3.0.0'
-    implementation 'com.google.firebase:firebase-datatransport:18.1.7'
+    implementation project(':firebase-datatransport')
```
Comment (eldhosembabu) [inline firebase-inappmessaging/firebase-inappmessaging.gradle:109]: @rlazo Please confirm this is okay. To use the new Transport annotation, i need to add the direct dependency on the data transport.

File: firebase-inappmessaging-display/CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # Unreleased
+* [changed] Fire log to clear cut migration.
```
Comment (rlazo) [inline firebase-inappmessaging-display/CHANGELOG.md:2]: I **think** it's a single word

File: firebase-inappmessaging/firebase-inappmessaging.gradle (hunk 1 above)
Comment (rlazo) [reply firebase-inappmessaging/firebase-inappmessaging.gradle:109]: Yes, WAI.

**Review by rlazo — APPROVED** [review]

File: firebase-inappmessaging-display/CHANGELOG.md (hunk 2 above)
Comment (eldhosembabu) [reply firebase-inappmessaging-display/CHANGELOG.md]: changed

**Pull request merged**